### PR TITLE
Landing page redesign

### PR DESCRIPTION
**Pull request by pedrogpimenta**
### Overview

Home, About and Community pages have been slightly redesigned with a lighter color scheme. Partners page has been removed and Partners have been added to the Home.

### Designs

https://www.figma.com/file/cZR0Uqhm8pf3G0uAJjRxMN/Marxan-Visual_V02-Copy?node-id=5460%3A7271

### Testing instructions

If it looks good, it should be good, this PR doesn't change much code.

### Feature relevant tickets

I'm sure there's one, but I don't know it :)

### CODE REVIEW

File: app/components/carousel/component.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const Carousel: React.FC<CarouselProps> = ({ images }: CarouselProps) =>
                 <div
                   className="relative w-full"
                   style={{
-                    paddingBottom: '56.25%',
+                    paddingBottom: '64.61318052%',
```
Comment (mbarrenechea) [inline app/components/carousel/component.tsx:85]: Why did you change this percentage? 56.25 is because 16/9 images. If this change is related to some reason I'm glad to hear it.

File: app/layout/home/carousel-section/component.tsx [hunk 2]
```diff
@@ -36,12 +38,34 @@ export const HomeCarousel: React.FC<HomeCarouselProps> = () => {
...
     <div
-      style={{
-        background: 'linear-gradient(to bottom, #000 75%, #151515 25%)',
-      }}
+      className="relative"
     >
+      <div
+        role="presentation"
+        style={{
+          backgroundImage: `url(${BackgroundImage})`,
```
Comment (mbarrenechea) [inline app/layout/home/carousel-section/component.tsx:46]: Is there any reason to not using tailwind here? I understand that you can set the background image inline but the rest I think you can do it in tailwind.

File: app/layout/home/features/component.tsx [hunk 3]
```diff
@@ -13,30 +13,37 @@ export interface HomeFeaturesProps {
...
-                <p className="text-gray-300">{description}</p>
-              </div>
-            );
-          })}
-          <div className="w-full mt-20 place-self-center">
-            <ButtonLink caption="Explore planning Examples" href="/community/projects" />
+              );
+            })}
+            <div className="w-full mt-20 place-self-center">
+              {/* <Link href="/community/projects">
```
Comment (mbarrenechea) [inline app/layout/home/features/component.tsx:36]: remove unnecessary commenteed code

File: app/layout/home/hero/component.tsx [hunk 4]
```diff
@@ -4,46 +4,63 @@ import Wrapper from 'layout/wrapper';
...
+            </div>
+          </div>
+          <div className="flex-shrink-0 relative top-14">
+            <img
+              alt=""
+              src={HERO_IMAGE}
+              style={{
+                width: '384px',
+                height: 'auto',
+                filter: 'drop-shadow(0px 8px 15px rgba(0, 0, 0, .35))',
```
Comment (mbarrenechea) [inline app/layout/home/hero/component.tsx:57]: mmmm, what if we add some cute animations for each image instead of adding these 4 images at once in one single image? We can talk about it

File: app/layout/header/component.tsx [hunk 5]
```diff
@@ -2,20 +2,24 @@ import React from 'react';
...
 import MyProjects from 'layout/header/my-projects';
 import Title from 'layout/header/title';
+import User from 'layout/header/user';
+import Wrapper from 'layout/wrapper';
 
 import LinkButton from 'components/button';
 
-import { useMe } from 'hooks/me';
-
+import LOGO_BLACK from 'svgs/logo-black.svg';
```
Comment (mbarrenechea) [inline app/layout/header/component.tsx:16]: Do we need a LOGO BLACK? maybe we can directly change the fill depending on the theme

File: app/layout/statics/button-link/component.tsx [hunk 6]
```diff
@@ -18,23 +19,23 @@ export const StaticButtonLink: React.FC<StaticButtonLinkProps> = ({
 }: StaticButtonLinkProps) => {
   const children = (
     <>
-      <p className="mr-4 text-lg text-primary-500 hover:underline">{caption}</p>
-      <div className="flex items-center justify-center bg-transparent border rounded-full h-7 w-7 border-primary-500">
-        <Icon icon={ARROW_RIGHT_2_SVG} className="w-3 h-3 text-primary-500" />
+      <p className="w-48 mr-4 text-lg text-gray-600 hover:text-gray-800 underline">{caption}</p>
```
Comment (mbarrenechea) [inline app/layout/statics/button-link/component.tsx:22]: I don't think adding a fixed width is the best option, If you put a maxWidth is better

File: app/layout/footer/component.tsx [hunk 7]
```diff
@@ -31,9 +31,6 @@ export const Footer: React.FC<FooterProps> = () => {
             <Link href="/community">
               <a href="/community" className="text-gray-400 hover:text-gray-200">Community</a>
             </Link>
-            <Link href="/partners">
```
Comment (mbarrenechea) [inline app/layout/footer/component.tsx:34]: I'm not 100% sure that we need to get rid of this page. I'll check it and let you know

**Review by mbarrenechea — CHANGES_REQUESTED** [review]
Let's have a quick chat about these comments tomorrow morning

File: app/components/carousel/component.tsx (hunk 1 above)
Comment (pedrogpimenta) [reply app/components/carousel/component.tsx]: Done

File: app/layout/home/carousel-section/component.tsx (hunk 2 above)
Comment (pedrogpimenta) [reply app/layout/home/carousel-section/component.tsx:47]: Done!

File: app/layout/home/features/component.tsx (hunk 3 above)
Comment (pedrogpimenta) [reply app/layout/home/features/component.tsx]: Done

File: app/layout/home/hero/component.tsx (hunk 4 above)
Comment (pedrogpimenta) [reply app/layout/home/hero/component.tsx:57]: I'll do it if there's time :)

File: app/layout/statics/button-link/component.tsx (hunk 6 above)
Comment (pedrogpimenta) [reply app/layout/statics/button-link/component.tsx:22]: I'll keep this to follow design. We'll change it if we need to.

Comment (pedrogpimenta) [reply app/layout/statics/button-link/component.tsx:22]: I'll keep this to follow design. We'll change it if we need to. (is this comment duplicated?)

**Review by mbarrenechea — APPROVED** [review]
